cp23/hpi-fun2/motivation/notes.md

```diff
@@ -0,0 +1,46 @@
+# Motivation
+
+The problem is about finding the path coverage of the tree. The problem should find the number of nodes, whos path lengths exceed the given threshold (including all its decendants). Also, it requires to find the number of paths to leafs.
+
+## Solution Idea
+
+> 1. Find all nodes, that where any parent (direct or in-direct) exceed the threshold, by traversion all paths top-down.
+> 2. Traverse the tree bottom-up and apply a DP structure, where:
+>
+>      ![DP Formula](./dp-formula.png)
```

Comment:
Oh, I just noticed you may have forgotten to commit this image.

Reply:
I see, thank you for spotting that :D